_rules/additional-content-triggered-on-hover-is-hoverable-ep1s13.md

```diff
@@ -0,0 +1,108 @@
+---
+id: ep1s13
+name: Additional content triggered on hover is hoverable
+rule_type: atomic
+description: |
+  This rule checks that any additional content that was triggered by hovering an element remains available to the user while the mouse pointer is over the element or the additional content
+accessibility_requirements:
+  wcag21:1.4.13: # Content on Hover or Focus (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Mouse pointer
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any element that when [hovered][] causes [visible changes in content][].
+
+## Expectation
+
+There are no [visible changes in content][] while the target element is [hovered][] or the mouse pointer remains inside a rectangle defined by the following coordinates:
+
+- (**top**:) the lowest value of the y coordinate of any pixel affected by the [visible changes in content][]; and
+- (**left**:) the lowest value of the x coordinate of any pixel affected by the [visible changes in content][]; and
+- (**bottom**:) the highest value of the y coordinate of any pixel affected by the [visible changes in content][]; and
+- (**right**:) the highest value of the x coordinate of any pixel affected by the [visible changes in content][].
+
+## Assumptions
```

Comment:
You're missing an important assumption; that any visible change triggers additional content to become visible. I don't think that's a safe assumption to make.

Reply:
I'm not sure I understand what you mean. If it's about the "additional" part, there is no requirement about it. The visible changes in content definition just requires that the rendered pixels change. That can be because new content is added, but also because existing content is changed.

If this was not what you were talking about, can you rephrase it?

Reply:
Your definition just means "visible change", not just in content, but anything really. If the color of a link on hover changes, this definition way make that a "visible change in content", while I'd argue it isn't a change in content at all. The problem is that animations on hover will fail this, because they constantly cause a visible change.

Reply:
What you're saying is correct, and it was done on purpose. I couldn't come up with a more effective (that would also be objective) way to detect changes in content. I don't know how we can address the problem of animations without using an assumption (which I had already included). 

But I do agree that this rule being applicable when hovering a link causes the link to change color or becoming underlined is too much. And that should not be taken care of with an assumption. So I've updated the applicability to exclude instances where the changes only happen inside the bounding box of the target element. In those instances it's impossible to have the pointer outside the element when it's over the tooltip anyway!